### PR TITLE
[WIP] Migration python3

### PR DESCRIPTION
**Pull request by benjello**

### CODE REVIEW

File: openfisca_france_data/dump.py [hunk 1]
```diff
@@ -1,39 +0,0 @@
...
-            if entity_data_frame is None:
-                entity_data_frame = data_frame
-            else:
-                entity_data_frame = pd.concat(
-                    [entity_data_frame, data_frame],
-                    axis = 1,
-                    )
-
-        print '======'
-    bim
```
Comment (maukoquiroga) [inline openfisca_france_data/dump.py]: That's probably why it didn't work!

**Comment by maukoquiroga** [comment]
Rebased

**Comment by maukoquiroga** [comment]
Rebased

**Comment by benjello** [comment]
Superseded by #161

**Pull request closed without merging**